pkg/generate/generate_test.go

```diff
@@ -13,19 +14,19 @@ import (
 
 type expected struct {
 	err        error
-	evaluation string
+	evaluation []string
 }
 
 func TestEvaluate(t *testing.T) {
 	for _, tt := range []struct {
 		name         string
-		template     string
+		file         io.Reader
 		combinations combination.Stream
 		expected     expected
 	}{
 		{
-			name:     "can process a template",
-			template: testdata.EvaluateInput,
+			name: "can process a template",
```

Comment:
should we also test error paths?

Reply:
For sure, I went ahead and added an error path check to the `TestBuildTemplate` function. Mind giving that look in [this commit](https://github.com/operator-framework/combo/pull/38/commits/6f59f42ed81e0592e459ae0ce3a809c06fec11f8)?